study-builder/studies/pancan/snippets/question-primary-cancer-list-child.conf

```
@@ -0,0 +1,65 @@
+{
+  include required("../../snippets/composite-question.conf"),
+  "stableId": ${id.q.primary_cancer_list_child},
+  "hideNumber": true,
+  "allowMultiple": true,
+  "childOrientation": "VERTICAL",
+  "promptTemplate": {
+    "templateType": "HTML",
+    "templateText": """
+              <p class="ddp-question-prompt">$prompt *<br/> <small><em>$note</em></small></p>
```

Since this is now a proper "prompt", we don't need the CSS class  or the `*`. So maybe something like this:
```suggestion
              $prompt <br/> <small><em>$note</em></small>
```